dataprofiler/profilers/profile_builder.py

```diff
@@ -1201,8 +1200,12 @@ def __add__(self, other):
         # merge profiles
         for profile_name in self._profile:
             merged_profile._profile[profile_name] = (
-                self._profile[profile_name] + other._profile[profile_name]
+                    self._profile[profile_name] + other._profile[profile_name]
```

Comment:
the previous pep8 changes make sense, but I'm not sure this should have two tabs.

Reply:
fixed